person.rb

```diff
@@ -9,10 +11,11 @@ def initialize(age, name = 'Unknown', parent_permission: true)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
   end
 
   attr_accessor :name, :age
-  attr_reader :id
+  attr_reader :id, :rentals
```

Comment:
- [ ] Kindly convert the `rentals` to `attr_accessor` as you already use it for writing in the Rental class initialize method.

Reply:
Ok, thanks for the clarification.